Dockerfile

```diff
@@ -1,9 +1,10 @@
 # Build the go application into a binary
 FROM golang:alpine as builder
 WORKDIR /app
-ADD . ./
+COPY . ./
 RUN CGO_ENABLED=0 GOOS=linux go build -mod vendor -a -installsuffix cgo -o gatus .
-RUN apk --update add ca-certificates
+RUN apk --update add ca-certificates upx
```

Comment:
I'd rather install packages after `FROM` to for caching.

Also, you could have `cgo` and `upx` commands in a single `RUN` to have a single layer.

Reply:
Sure thing, though it makes no noticeable difference since the resulting image is a using `scratch` as base. And I was trying not nitpick, and follow the current pattern as much as possible.

Imho, it's best to install OS packages separately from "go build", but since it's only `ca-certificates` sure - multiple RUN statements can be merged into one.

Moving package installation to after builder's `FROM`